central/complianceoperator/v2/compliancemanager/manager_impl.go

```diff
@@ -219,6 +222,20 @@ func (m *managerImpl) processRequestToSensor(ctx context.Context, scanRequest *s
 		return nil, errors.Wrapf(err, "Unable to create scan configuration named %q.", scanRequest.GetScanConfigName())
 	}
 
+	// Check if there are any Existing ScanSettingBindings for the clusters that already have the scan configuration
+	// If so, then we cannot create the scan configuration.
+	existingScanSettingBindings, err := m.ssbDatastore.GetScanSettingBindings(ctx, search.NewQueryBuilder().
+		AddExactMatches(search.ClusterID, clusters...).ProtoQuery())
+	if err != nil {
+		return nil, errors.Wrapf(err, "Unable to retrieve scan setting bindings for scan configuration named %q.", scanRequest.GetScanConfigName())
```

Comment:
If a customer creates a scan configuration called `foo` that uses profiles `ocp4-cis` and `ocp4-cis-node`, then creates a second scan configuration called `bar` that uses profiles `ocp4-cis` and `ocp4-high`, and for some reason if there is an error here, it would be reported as:

  "Unable to retrieve scan setting bindings for scan configuration named bar."

Do we want to generalize this error message? The scan setting bindings don't exist for the second scan configuration, yet.

Reply:
Yeah we are getting the scan setting bindings for the clusters in the configuration.  This error just needs to be more generic.